backend/app/categories/controller.py

```diff
@@ -17,9 +18,23 @@
 async def get_category_route(category_id: str,
```

Comment:
This function does not has a response model set. This should be added

Reply:
If you add that. After that i dont think you will be able to send ItemsInResponse as there data is required. And you dont send any data. This can be fixed by raising a HTTPExeption instead of returning

Reply:
Line 17, ```response_model=ItemsInResponse``` ?

Reply:
in the decrator router.get line

Reply:
@kjain1810 set `response_model=ItemsInResponse` and that's it. There's none of that HTTPException stuff required.